multipaz/src/commonMain/kotlin/org/multipaz/statuslist/StatusListBuilder.kt

```diff
@@ -0,0 +1,52 @@
+package org.multipaz.statuslist
+
+import kotlinx.io.Buffer
+import kotlinx.io.readByteArray
+
+class StatusListBuilder(val bitsPerItem: Int) {
```

Comment:
Usually we put builders in the thing they build, suggest to move to StatusList.

We also usually do `fun buildStatusList(...): StatusList` which takes builder constructor params and also builderAction.